docusaurus/website/siteConfig.js

```diff
@@ -0,0 +1,84 @@
+/**
+ * Copyright (c) 2017-present, Facebook, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+// See https://docusaurus.io/docs/site-config for all the possible
+// site configuration options.
+
+const siteConfig = {
+  title: "React Redux", // Title for your website.
+  tagline: "React Redux Documentation",
+  url: "https://your-docusaurus-test-site.com", // Your website URL
+  baseUrl: "/", // Base URL for your project */
+  // For github.io type URLs, you would set the url and baseUrl like:
+  //   url: 'https://facebook.github.io',
+  //   baseUrl: '/test-site/',
+
+  // Used for publishing and more
+  projectName: "react-redux",
+  organizationName: "facebook",
+  // For top-level user or org sites, the organization is still the same.
+  // e.g., for the https://JoelMarcey.github.io site, it would be set like...
+  //   organizationName: 'JoelMarcey'
+
+  // For no header links in the top nav bar -> headerLinks: [],
+  headerLinks: [
+    { doc: "getting-started", label: "Getting started" },
+    { doc: "api", label: "Api" },
+    { doc: "troubleshooting", label: "Troubleshooting" }
+  ],
+
+  /* path to images for header/footer */
+  // headerIcon: "img/redux.svg",
+  // footerIcon: "img/redux.svg",
+  favicon: "img/favicon/favicon.ico",
+
+  /* Colors for website */
+  colors: {
+    primaryColor: "#20232a",
```

Comment:
Seems like these are shades of blue?  Let's try to go with Redux's purple and white, or something like that.